benchmarks/makefile

```diff
@@ -0,0 +1,11 @@
+.PHONY: default
+
+default:
```

Comment:
This should be `all`.

Reply:
This isn't actually an "all" target. I renamed it to something that is hopefully clearer.